Controller/BackupController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+
+namespace CustomerBackupAndDelete\Controller;
+
+
+use Thelia\Controller\Admin\BaseAdminController;
+use Thelia\Model\CustomerQuery;
+use Thelia\Tools\URL;
+
+class BackupController extends BaseAdminController
+{
+    public function transferOrders($id) {
+        $customer = CustomerQuery::create()->findOneById($id);
+        $backupCustomer = CustomerQuery::create()->findOneByRef('BACKUPORDERS');
+
+        if (null !== $orders = $customer->getOrders()) {
```

Comment:
I think this missing check if backupCustomer exist and maybe create it if not exist